File: src/files/CheckInput.py
def isNumber(s):

    for i in range(len(s)):
        if s[i].isdigit() != True:
            return False
    return True

if __name__ == "__main__":
    str = "6790"
    if isNumber(str):
        print("Integer")
    else:
        print("String")

File: src/files/CheckInput.cpp
#include <iostream>
using namespace std;

bool isNumber(string s)
{
	for (int i = 0; i < s.length(); i++)
		if (isdigit(s[i]) == false)
			return false;
	return true;
}

int main()
{
	string str = "6790";
	if (isNumber(str))
		cout << "Integer";
	else
		cout << "String";
}